Update microsoft-defender-endpoint-android.md (#6127)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/defender-endpoint/microsoft-defender-endpoint-android.md

    -   Access to the Microsoft 365 Defender portal.

        > [!NOTE]

>         > Microsoft Defender for Endpoint can now extends the protection to organization’s data within a managed application for those who aren’t using an MDM but are using Intune to manage mobile applications. It also extends this support to customers who use other enterprise mobility management solutions, while still using Intune for [Mobile Application Management (MAM).](https://docs.microsoft.com/en-us/mem/intune/apps/mam-faq)

Reviewer: ```suggestion
        > Microsoft Defender for Endpoint can now extend the protection to an organization’s data within a managed application for those who aren’t using mobile device management (MDM) but are using Intune to manage mobile applications. It also extends this support to customers who use other enterprise mobility management solutions, while still using Intune for [mobile application management (MAM)](/mem/intune/apps/mam-faq).
```

> >Prior to this update, Microsoft Defender for Endpoint worked on devices that were enrolled using Intune mobile device management (MDM) only.  

Reviewer: ```suggestion
         > Prior to this update, Microsoft Defender for Endpoint worked on devices that were enrolled using Intune MDM only.  
```

## Related topics
- [Deploy Microsoft Defender for Endpoint on Android with Microsoft Intune](android-intune.md)
- [Configure Microsoft Defender for Endpoint on Android features](android-configure.md)

> - >[What is mobile app management in Microsoft Intune? | How to get started with MAM.](https://docs.microsoft.com/en-us/mem/intune/apps/app-management#mobile-application-management-mam-basics)

Reviewer: ```suggestion
- [Mobile Application Management (MAM) basics](/mem/intune/apps/app-management#mobile-application-management-mam-basics)
```
